jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/propertywindow.js

```diff
@@ -6037,13 +6056,13 @@ Ext.form.ConditionExpressionEditorField = Ext.extend(Ext.form.TriggerField,  {
             }]
         });
 
-        function initCodeEditor() {
+        function initCodeEditor(myTypeMode, myShowLineNumbers, myDoLineWrapping, myDoMatchBrackets) {
             this.foldFunc = CodeMirror.newFoldFunction(CodeMirror.braceRangeFinder);
             sourceEditor = CodeMirror.fromTextArea(document.getElementById(scriptEditor.getId()), {
-                mode: "text/x-java",
-                lineNumbers: true,
-                lineWrapping: true,
-                matchBrackets: true,
+                mode: myTypeMode,
+                lineNumbers: myShowLineNumbers,
+				lineWrapping: myDoLineWrapping,
```

Comment:
It looks like tabulation sign usage. @tsurdilo you can disable "Use tab character" and IDEA will replace Tabs by whitespaces automatically. To do it: Settings > Editor > Code Style > Java > Tabs and Indents > and uncheck "Use tab character"